src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -906,6 +913,42 @@ private static bool AddFlags(TypeSymbol type, ArrayBuilder<bool> transformFlagsB
 
                 // Continue walking types
                 return false;
+
+                static void handleFunctionPointerType(FunctionPointerTypeSymbol funcPtr, ArrayBuilder<bool> transformFlagsBuilder, bool isNestedNamedType, bool addCustomModifierFlags)
```

Comment:
>isNestedNamedType [](start = 136, length = 17)

Do we have a test involving nested named types in a function pointer?

Reply:
We cannot, but it does mean I need to find a test that breaks the bug I have.

---
In reply to: [422328571](https://github.com/dotnet/roslyn/pull/44078#discussion_r422328571) [](ancestors = 422328571)

Reply:
> We cannot, but it does mean I need to find a test that breaks the bug I have.

The parameter looks unused to me in iteration 4

---
In reply to: [422419465](https://github.com/dotnet/roslyn/pull/44078#discussion_r422419465) [](ancestors = 422419465,422328571)

Reply:
Exactly. There's a bug, I need to find a test.

---
In reply to: [422530525](https://github.com/dotnet/roslyn/pull/44078#discussion_r422530525) [](ancestors = 422530525,422419465,422328571)

Reply:
Talking with Aleksey, this should never be true for function pointer types. I'll assert it's false.

---
In reply to: [423322684](https://github.com/dotnet/roslyn/pull/44078#discussion_r423322684) [](ancestors = 423322684,422530525,422419465,422328571)

Reply:
> Talking with Aleksey, this should never be true for function pointer types. I'll assert it's false.

I suggest asserting at the call site and removing the parameter here.

---
In reply to: [423336064](https://github.com/dotnet/roslyn/pull/44078#discussion_r423336064) [](ancestors = 423336064,423322684,422530525,422419465,422328571)